site/gatsby-site/src/global.css

```diff
@@ -53,8 +53,15 @@ a {
   text-decoration: none !important;
 }
 
-li {
-  list-style-position:inside;
+ol, ul {
```

Comment:
This changes all the lists on the site, I would say you should create custom components and add them here: https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/src/components/ui/MdxComponents.js

Reply:
Or maybe we should be using this https://flowbite.com/docs/components/typography/ instead?

Reply:
This a solution to the general problem in #1326, included in this PR for now because the page is unreadable without a fix. It restores the default style that got overridden with the introduction of Tailwind. There is no reason for this to be a custom component – it's a completely vanilla list item.

This particular page has heavily nested lists, which Flowbite Typography [might not support and isn't designed for](https://flowbite.com/plugins/typography/#:~:text=i'm%20not%20sure%20if%20we'll%20bother%20styling%20more%20than%20two%20levels%20deep.).

Reply:
We need to let tailwind/flowbite overwrite vanilla styles and work from there. This way, any developer that copies/pastes elements from Flowbite's docs will have them working out of the box. If we undo these overrides, this stops working.

I mentioned the custom component because it was the fastest way to deal with this, but ideally, I think we should be using Flowbite's typography defaults here, too; it should be just a matter of setting the `prose` class at the top of our markdown-based content, and that should be it.

```
<article class="prose lg:prose-xl">
  {{ markdown }}
</article>
```  

Are you sure it will look too bad?

Reply:
To add more context, if we undo global styles, we defeat the purpose of this: https://tailwindcss.com/docs/preflight